Userland/Applications/CharacterMap/CharacterMapWidget.cpp

```diff
@@ -119,6 +123,24 @@ CharacterMapWidget::CharacterMapWidget()
         GUI::Clipboard::the().set_plain_text(m_output_box->text());
     };
 
+    m_unicode_block_listview->on_activation = [this](auto& index) {
+        if (index.row() > 0)
+            m_range = Unicode::g_block_display_names[index.row() - 1].code_point_range;
```

Comment:
Ah I see, this is where the extern var is used - so we can't rely on this existing either. We can instead have a weakly linked function to access this data, and generate a strong definition when `ENABLE_UNICODE_DATABASE_DOWNLOAD` is enabled

Reply:
Made a weak function instead of using the extern and moved the needed structs into CharacterTypes.h. Hopefully no reliance on UnicodeData.h anymore. Thanks for the help so far